src/charm.py

```diff
@@ -288,6 +308,20 @@ def _on_ingress_revoked(self, event: IngressPerAppRevokedEvent) -> None:
             logger.info("This app no longer has ingress")
             self.oauth.update_client_config(client_config=self._oauth_client_config)
 
+    @log_event_handler(logger)
+    def _on_trusted_certificates_available(self, event: CertificatesAvailableEvent) -> None:
+        self._handle_status_update_config(event)
+
+    @log_event_handler(logger)
+    def _on_trusted_certificates_removed(self, event: CertificatesRemovedEvent) -> None:
+        self._container.remove_path(CERTIFICATES_FILE)
```

Comment:
question: is this needed? Won't this be overwritten by `update_trusted_ca_certs`?

Reply:
Good point, fixed.